casper-tool.py

```diff
@@ -350,7 +350,7 @@ def create_network(
                       node_version, public_address, None)
         validator_keys.append(account)
 
-    initial_known_nodes = bootstrap_nodes + validator_nodes
+    initial_known_nodes = bootstrap_nodes
```

Comment:
This was changed to match the changes in 

https://github.com/dwerner/casper-test-ansible/blob/48179ed3b26e4c57a8902926b719b5126c9c4d67/casper-tool.py#L353